main.css

```diff
@@ -0,0 +1,50 @@
+@import url('https://fonts.googleapis.com/css?family=Merriweather:900|Roboto');
+
+* {
+    width: 625px;
+    color: white;
+}
+
+span {
+    border: 1px rgba(211, 211, 211, 0.2);
```

Comment:
`border: 1px rgba(211, 211, 211, 0.2);` this line seems unnecessary. If you don't want visible borders you can put it to border: none or border-color: transparent.